File: 6 - Let's Review/Solution.cs
using System;

class Solution
{
    static void Main(String[] args)
    {
        var N = int.Parse(Console.ReadLine());

        for (var i = 0; i < N; i++)
        {
            var str = Console.ReadLine();

            for (var j = 0; j < str.Length; j++)
            {
                if (j % 2 == 0) Console.Write(str[j]);
            }

            Console.Write(" ");

            for (var j = 0; j < str.Length; j++)
            {
                if (j % 2 != 0) Console.Write(str[j]);
            }

            Console.Write(Environment.NewLine);
        }
    }
}

File: 6 - Let's Review/Solution.fs
open System

[<EntryPoint>]
let main argv = 
    let n = Console.ReadLine() |> int
    for i = 0 to n - 1 do
        let str = Console.ReadLine()
        for j = 0 to str.Length - 1 do
            if j % 2 = 0 then Console.Write(str.[j])
        Console.Write(" ")
        for j = 0 to str.Length - 1 do
            if j % 2 <> 0 then Console.Write(str.[j])
        Console.Write(Environment.NewLine)
    0
